openreview/conference/templates/areachairWebfield.js

```diff
@@ -827,6 +856,17 @@ var registerEventHandlers = function() {
       promptError('Reviewer ' + view.prettyId(userToAdd) + ' has already been assigned to Paper ' + paperNumber.toString());
       return false;
     }
+
+    var reviewersWithConflict = paperAndReviewersWithConflict[paperNumber];
+    if (!ENABLE_REVIEWER_REASSIGNMENT_TO_OUTSIDE_REVIEWERS && //if enabled then skip checking
+      (_.includes(allReviewers, userToAdd) == false) || //input is not in dropdown or 
```

Comment:
Please change this to `=== false` or `!_.includes`. We try to avoid using double equals.